molecule/amqp_default_interval/verify.yml

```diff
@@ -0,0 +1,18 @@
+---
+- import_playbook: ../common/verify.yml
+
+- name: Verify
+  hosts: collectd-test
+  tasks:
+    - name: "Get amqp conf"
+      stat:
+        path: "{{ collectd_conf_output_dir }}/amqp1.conf"
+      register: amqp_conf
+      failed_when: "{{ not amqp_conf.stat.exists }}"
+
+    - name: "Check that default interval was unchanged"
+      command:
+        grep "Interval" {{ collectd_conf_output_dir }}
+      register: interval
+      failed_when:
+        - "{{ interval.stdout | length != 0 }}"
```

Comment:
:-1: I don't think this test works. I think that `grep "Interval" /etc/collectd.d/` will always return a length 0 stdout, and the following on stderr: `grep: /etc/collectd.d/: Is a directory`. I think you mean to add a `-R` to your grep?

IIUC this test is attempting to verify that no "Interval" directive appears if we've specified an invalid value for collectd_amqp_interval?

Reply:
Oh wow, this is a big boo boo. 
It should be checking for Interval in {{ amqp_conf }}, to make sure that the global Interval is not being overridden by a plugin-specific Interval, when the CollectdAmqpInterval is set to -666 (i.e. default).

(See above for links)